lib/telemetry/armon_metrics.go

```diff
@@ -242,36 +287,36 @@ func (c *TelemetryConfig) MergeDefaults(defaults *TelemetryConfig) {
 	}
 }
 
-func statsiteSink(cfg TelemetryConfig, hostname string) (metrics.MetricSink, error) {
+func statsiteSink(cfg Config) (metrics.MetricSink, error) {
 	addr := cfg.StatsiteAddr
 	if addr == "" {
 		return nil, nil
 	}
 	return metrics.NewStatsiteSink(addr)
 }
 
-func statsdSink(cfg TelemetryConfig, hostname string) (metrics.MetricSink, error) {
+func statsdSink(cfg Config) (metrics.MetricSink, error) {
 	addr := cfg.StatsdAddr
 	if addr == "" {
 		return nil, nil
 	}
 	return metrics.NewStatsdSink(addr)
 }
 
-func dogstatdSink(cfg TelemetryConfig, hostname string) (metrics.MetricSink, error) {
+func dogstatsdSink(cfg Config) (metrics.MetricSink, error) {
 	addr := cfg.DogstatsdAddr
 	if addr == "" {
 		return nil, nil
 	}
-	sink, err := datadog.NewDogStatsdSink(addr, hostname)
+	sink, err := datadog.NewDogStatsdSink(addr, "")
```

Comment:
Is this a breaking change? Does it materially change the metrics that get sent to datadog?

Reply:
I think the answer is no because we never exposed setting a hostname for metrics explicitly.

Reply:
This should not be a breaking change -- go-metrics sets the hostname tag without needing to be provided one at the sink level.